grid-column-span-with-sibling-count/css/main.css

```
@@ -0,0 +1,47 @@
+  &.compact {
+    --min-col-span: 1;
+    --max-col-span: 4;
+  }
+
+  &.spacious {
+    --min-col-span: 4;
+    --max-col-span: 8;
+  }
+
+  li {
+    --card-count: sibling-count();
```

Biome (2.1.2)

[error] Unexpected unknown function: sibling-count

Use a known function instead.
See MDN web docs for more details.

(lint/correctness/noUnknownFunction)